data/strings/strings.txt

```diff
@@ -5256,6 +5256,24 @@
     zh-Hans = 语音语言
     zh-Hant = 語音語言
 
+  [pref_tts_test_voice_title]
+    comment = Settings «Route» category: «Tts Test Voice» title
+    tags = android
+    en = Test Text-To-Speech (TTS) Voice
+    es = Probar síntesis de voz
+    nl = Test stem
+    pt = Testar voz
+    pt-BR = Testar voz
+
+  [pref_tts_playing_test_voice]
+    comment = Settings «Route» category: «Playing Voice Test» pop-up message
+    tags = android
+    en = Playing voice test
```

Comment:
This message will also hint to a user what should be done if it doesn't work:
```
    en = Check the volume or system Text-To-Speech settings if you don't hear the voice now.
    ar = تحقق من مستوى الصوت أو إعدادات تحويل النص إلى كلام في النظام إذا كنت لا تسمع الصوت الآن.
    be = Праверце гучнасць або сістэмныя налады пераўтварэння тэксту ў маўленне, калі вы зараз не чуеце голас.
    bg = Проверете силата на звука или системните настройки за преобразуване на текст в реч, ако сега не чувате гласа.
    ca = Comproveu el volum o la configuració de text a veu del sistema si ara no escolteu la veu.
    cs = Pokud nyní hlas neslyšíte, zkontrolujte nastavení hlasitosti nebo převodu textu na řeč v systému.
    da = Tjek lydstyrken eller systemets tekst-til-tale-indstillinger, hvis du ikke kan høre stemmen nu.
    de = Überprüfe die lautstärke oder die text-to-speech-einstellungen des systems, wenn du die stimme jetzt nicht hörst.
    el = Ελέγξτε την ένταση ήχου ή τις ρυθμίσεις του συστήματος text-to-speech αν δεν ακούτε τη φωνή τώρα.
    es = Comprueba el volumen o la configuración de texto a voz del sistema si ahora no oyes la voz.
    es-MX = Verifique el volumen o la configuración de texto a voz del sistema si no escucha la voz ahora.
    et = Kontrollige helitugevust või süsteemi teksti kõnetamise seadeid, kui te ei kuule nüüd häält.
    eu = Egiaztatu bolumena edo sistemaren testu-ahotsaren ezarpenak ahotsa entzuten ez baduzu.
    fa = اگر اکنون صدا را نمی شنوید، میزان صدا یا تنظیمات تبدیل متن به گفتار سیستم را بررسی کنید.
    fi = Tarkista äänenvoimakkuus tai järjestelmän tekstistä puheeksi -asetukset, jos ääni ei kuulu nyt.
    fr = Vérifie le volume ou les paramètres de synthèse vocale du système si tu n'entends pas la voix maintenant.
    he = בדוק את עוצמת הקול או את הגדרות הטקסט לדיבור של המערכת אם אינך שומע את הקול כעת.
    hi = यदि आपको अभी आवाज नहीं सुनाई देती है तो वॉल्यूम या सिस्टम टेक्स्ट-टू-स्पीच सेटिंग्स की जांच करें।
    hu = Ellenőrizze a hangerőt vagy a rendszer szöveg-beszéd beállításait, ha most nem hallja a hangot.
    id = Periksa pengaturan volume atau pengaturan teks-ke-ucapan sistem jika anda tidak mendengar suara sekarang.
    it = Controlla il volume o le impostazioni del sistema di sintesi vocale se ora non senti la voce.
    ja = 音声が聞こえない場合は、音量またはシステムの音声合成設定を確認する。
    ko = 지금 음성이 들리지 않으면 볼륨 또는 시스템 텍스트 음성 변환 설정을 확인하세요.
    mr = तुम्हाला आता आवाज ऐकू येत नसल्यास व्हॉल्यूम किंवा सिस्टम टेक्स्ट-टू-स्पीच सेटिंग्ज तपासा.
    nb = Kontroller volumet eller systeminnstillingene for tekst-til-tale hvis du ikke hører stemmen nå.
    nl = Controleer het volume of de tekst-naar-spraak instellingen van het systeem als je de stem nu niet hoort.
    pl = Jeśli nie słyszysz głosu, sprawdź głośność lub systemowe ustawienia zamiany tekstu na mowę.
    pt = Verifica o volume ou as definições de conversão de texto em voz do sistema se não ouvires a voz agora.
    pt-BR = Verifique o volume ou as configurações de conversão de texto em fala do sistema se você não estiver ouvindo a voz agora.
    ro = Dacă nu auziți vocea acum, verificați volumul sau setările sistemului text-to-speech.
    ru = Проверьте громкость или системные настройки преобразования текста в речь, если сейчас ничего не слышно.
    sk = Ak teraz nepočujete hlas, skontrolujte hlasitosť alebo systémové nastavenia prevodu textu na reč.
    sv = Kontrollera volymen eller systemets text-till-tal-inställningar om du inte hör rösten nu.
    sw = Angalia mipangilio ya sauti au ya mfumo ya kubadilisha maandishi-hadi-hotuba ikiwa husikii sauti sasa.
    th = ตรวจสอบระดับเสียงหรือการตั้งค่าการอ่านออกเสียงข้อความของระบบหากคุณไม่ได้ยินเสียงในตอนนี้
    tr = Sesi şimdi duymuyorsanız ses seviyesini veya sistem metin okuma ayarlarını kontrol edin.
    uk = Якщо ви не чуєте голос, перевірте гучність або налаштування системи перетворення тексту на мовлення.
    vi = Kiểm tra cài đặt âm lượng hoặc chuyển văn bản sang giọng nói của hệ thống nếu bây giờ bạn không nghe thấy giọng nói.
    zh-Hans = 如果现在听不到语音，请检查音量或系统文本到语音设置。
    zh-Hant = 如果您現在聽不到聲音，請檢查音量或系統文本轉語音設置。
```

Reply:
Small correction for german:
`de = Überprüfe die Lautstärke oder die Text-to-speech Einstellungen des Systems, wenn du die Stimme jetzt nicht hörst.`

Reply:
Thanks @biodranik, @Mr-Mime... I've amended the commit with these messages...

Due to the longer size of the strings, I had to replace the `Toast` message-box with a `Snackbar` message-box:

![snackbar](https://github.com/organicmaps/organicmaps/assets/25888296/a7a5e9fc-aecc-4fd7-a808-105ae31fffa1)